addons/web/static/src/views/widgets/documentation_link/documentation_link.scss

```diff
@@ -0,0 +1,3 @@
+.o_widget_documentation_link {
+    display: contents;
```

Comment:
not sure about this. In fields, we can use additionalClasses to add class on the parent node. Would the same feature for widgets allow you to get rid of this rule?